examples/features/nse-composition/passthrough-1/passthrough1.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: nse-firewall-vpp
+  name: nse-passthrough-1
+spec:
+  selector:
+    matchLabels:
+      app: nse-firewall-vpp
+  template:
+    metadata:
+      labels:
+        app: nse-firewall-vpp
+    spec:
+      containers:
+      - env:
+        - name: NSM_SERVICE_NAME
+          value: nse-composition
+        - name: NSM_LABELS
+          value: app:passthrough-1
+        - name: SPIFFE_ENDPOINT_SOCKET
+          value: unix:///run/spire/sockets/agent.sock
+        - name: NSM_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: NSM_CONNECT_TO
+          value: unix:///var/lib/networkservicemesh/nsm.io.sock
+        image: ghcr.io/networkservicemesh/ci/cmd-nse-firewall-vpp:27f0c90
```

Comment:
Could you rearrange the items?
All of our deployments (or at least the vast majority) have the following order:
```yaml
containers:
  image: ...
  imagePullPolicy: ...
  env: ...
  volumeMounts: ...
  resources: ...
```
It would be easier to read if your example was also using this order.

Same for all the other deployments.

Reply:
ok, done